app/app.go

```diff
@@ -761,7 +764,7 @@ func New(
 	app.RollappKeeper.SetHooks(rollappmoduletypes.NewMultiRollappHooks(
 		// insert rollapp hooks receivers here
 		app.SequencerKeeper.RollappHooks(),
-		transferStack.(delayedackmodule.IBCMiddleware),
+		delayedAckMiddleware,
```

Comment:
`transferStack`

Reply:
what's the difference between `transferStack.(delayedackmodule.IBCMiddleware)` and `delayedAckMiddleware` ?

Reply:
my bad